domain-model/src/test/java/uk/gov/hmcts/cmc/domain/models/claimantresponse/ResponseAcceptationTest.java

```diff
@@ -25,38 +24,12 @@ public void shouldBeSuccessfulValidationForValidResponse() {
         assertThat(response).hasSize(0);
     }
 
-    @Test
-    public void shouldBeInvalidWhenAmountNotPresent() {
-        ClaimantResponse claimantResponse = ResponseAcceptation.builder()
-            .amountPaid(null)
-            .formaliseOption(REFER_TO_JUDGE)
-            .determinationDecisionType(DEFENDANT)
-            .build();
-
-        Set<String> response = validate(claimantResponse);
-
-        assertThat(response).hasSize(1);
-    }
-
-    @Test
-    public void shouldBeInvalidWhenAmountIsNegative() {
```

Comment:
I bet that should stay. Negative amounts should be forbidden.

Reply:
agree